packages/date-picker/src/FilterDateRangePicker/components/DateRangeInputField/DateRangeInputField.module.scss

```diff
@@ -0,0 +1,55 @@
+@import "~@kaizen/design-tokens/sass/border";
+
+$input-disabled-opacity: 0.3;
+
+.dateRangeInputContainer {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  border: none;
+}
+
+.legend {
+  clip: rect(0 0 0 0);
+  clip-path: inset(50%);
+  height: 1px;
+  overflow: hidden;
+  position: absolute;
+  white-space: nowrap;
+  width: 1px;
+}
+
+.inputRangeStart {
+  border-start-end-radius: 0;
+  border-end-end-radius: 0;
+
+  &:focus + div {
+    border-start-end-radius: 0;
+    border-end-end-radius: 0;
+  }
+}
+
+.inputRangeEnd {
+  position: relative;
+  inset-inline-start: calc(-1 * #{$border-solid-border-width});
+
+  border-start-start-radius: 0;
+  border-end-start-radius: 0;
+
+  &:focus + div {
+    border-start-start-radius: 0;
+    border-end-start-radius: 0;
+
+    // Override position of focus ring
+    /* stylelint-disable-next-line declaration-no-important */
```

Comment:
I know this is a smell, but unless I refactor the Input component, I can't do anything about it :'(